jasync-sql/src/test/groovy/io/micronaut/configuration/jasync/JasyncConfigurationSpec.groovy

```diff
@@ -52,6 +84,26 @@ class JasyncConfigurationSpec extends Specification {
         applicationContext?.stop()
     }
 
+    //
+    void "test jasync-client interceptors configuration"() {
+        given:
+        ApplicationContext applicationContext = ApplicationContext.run(
+                'jasync.client.port': '5433'
+        )
 
+        when:
+        applicationContext.registerSingleton(new DummyQueryInterceptor1())
+        applicationContext.registerSingleton(new DummyQueryInterceptor2())
```

Comment:
Can you replace this with usage of spock mocks / stubs. It will be easier to maintain.

Reply:
sure sure

Reply:
To demonstrate that list of interceptors will be injected I needed to create two stubs, unfortunately registering two stubs of type QueryInterceptor will result into having one bean in AppContext, so instead I've used existing jasync QueryInterceptors. With that it even more demonstrates (tests) the functionality.